server/email/email-router.js

```diff
@@ -0,0 +1,21 @@
+var express = require('express');
+var path = require('path');
+
+module.exports = function(app, io){
+  app.use(express.static(__dirname));
+  var emailRouter = express.Router();
+
+  var emailController = require('./email-controller')(app, io);
+
+
+  emailRouter.post('/confirmation', emailController.sendEmails);
+
+  emailRouter.get('/confirmation/confirmation.html', function(req,res){
+    res.sendFile(path.join(__dirname, './confirmation.html'));
+  });
+
+  emailRouter.get('/confirmation/response*', emailController.handleResponse);
```

Comment:
Yes because the url of this get request has information like eventId, userId, confirmation. So I need to grab the whole thing no matter what comes after /confirmation/response